index.bs

```diff
@@ -279,5 +512,149 @@ Threat model {#security-threat}
 Mitigations and defenses {#security-defenses}
 ---------------------------------------------
 
+Appending A: Messages
+=====================
+
+Lower type keys are used for messages we expect to be more frequent
+and higher type keys are used for less frequently used messages.  This
+is because on the wire lower values encode smaller.
+
+<pre>
+; type key 14
```

Comment:
We could probably hack up the IDL formatter in bikeshed to pretty print these in the spec.

Reply:
Do you want a TODO for that?

Reply:
Sure.

Reply:
I added a TODO